mathesar/urls.py

```diff
@@ -28,8 +28,7 @@
     path('api/ui/v0/', include(ui_router.urls)),
 
     path('', views.home, name="home"),
-    path('<db_name>/', views.db_home, name="db_home"),
-    path('<db_name>/schemas/', views.schemas, name="schemas"),
+    path('<db_name>/', views.schemas, name="schemas"),
```

Comment:
There is no `schemas` page in the FE. The DB page and schemas pages are the same pages hence made the single route using the `schemas` view.